src/resolve/resolved-dns-packet.c

```diff
@@ -1653,11 +1653,11 @@ int dns_packet_read_key(
                 return r;
 
         if (p->protocol == DNS_PROTOCOL_MDNS) {
-                /* See RFC6762, Section 10.2 */
+                /* See RFC6762, sections 5.4 and 10.2 */
 
                 if (type != DNS_TYPE_OPT && (class & MDNS_RR_CACHE_FLUSH)) {
```

Comment:
hmm, MDNS_RR_CACHE_FLUSH is a bit of a misnomer now. We should either rename that to maybe MDNS_RR_CACHE_FLUSH_OR_QU or so?

Reply:
Yes! I was unsure what to rename it to, either the `CACHE_FLUSH_OR_QU` thing as you suggest, or just `TOP_BIT` (I went with the latter for the variable). Should I name both `cache_flush_or_qu` then?

Reply:
yeah, i think that would be better